css/style.css

```diff
@@ -1 +1,9 @@
+body{font-family: Geneva;
```

Comment:
with css styles, it's good to leave a space between the selector (body) and the {. Also, hit return here to put font-family on the next line. This doesn't affect the web page but people you work with will like you a lot more because your code is easier to read.